vendor/VRControls.js

```diff
@@ -35,9 +35,9 @@ THREE.VRControls = function ( object, onError ) {
 
 	}
 
-	if ( navigator.getVRDisplays ) {
+	if ( window.navigator.getVRDisplays ) {
```

Comment:
Without that change, I get this stack trace:

```
  1) node acceptance tests can run in node:
     ReferenceError: navigator is not defined
      at new THREE.VRControls (vendor\VRControls.js:38:7)
      at Object.<anonymous> (src\utils\device.js:3:16)
      at require (internal/module.js:11:18)
      at Object.<anonymous> (src\utils\index.js:5:14)
      at require (internal/module.js:11:18)
      at Object.<anonymous> (src\index.js:1:75)
      at require (internal/module.js:11:18)
      at Context.<anonymous> (tests\node\test.js:18:18)
```

So it may currently be incorrectly run at require time?